qualtran/__init__.py

```diff
@@ -53,6 +53,7 @@
     QUInt,
     BQUInt,
     QMontgomeryUInt,
+    QGF,
```

Comment:
any concerns about effects on import time? I know nothing will be slower than quimb but you can theoretically avoid importing quimb sometimes

Reply:
Moved `from galois import GF` to a local import inside a method of `QGF`. Verified that `time python -c 'import qualtran'` has no degradation from this PR now.